### ISSUE TITLE
New Launcher Icon?

### ISSUE THREAD

**Issue by DavidXochipilli** [oliexdev/openScale#67]
Hey @oliexdev,
thanks you so much for your awesome app! Are you interested in a new launcher icon?
I am not good at design myself but I could ask someone. Do you have any wishes?

**Comment by oliexdev** [comment]
Hi DavidXochipilli,

I am very interested in a new launcher icon. :smiley: I am not a good graphic designer as well. 
I have only two wishes: Firstly the icon launcher should be available in the following sizes to met the android design requirements:
512x512px,  192x192px, 144x144px, 96x96px, 72x72px, 48x48px or in a svg format.
Secondly the icon must be licensed in a way that I can use it in my app, see for example https://creativecommons.org/ for some licenses (the authors name will be of course included in the CREDITS)
Just for your information Google has published a design guide for icons https://material.io/guidelines/style/icons.html#icons-product-icons maybe it helps a bit.
My used icon colour is `#33b6ea` which should be somehow used in the new icon. 

Best regards from Germany
oli

**Comment by DavidXochipilli** [comment]
@Sajidshaik will do the design and I will send him a few dollars for his work 👌

**Comment by DavidXochipilli** [comment]
Here's a preview @oliexdev, please tell us your thoughts:
![openscale_materialicon_preview](https://user-images.githubusercontent.com/29784866/29787684-1a3e4a30-8c30-11e7-8fa2-61cb6e30898f.png)

**Comment by oliexdev** [comment]
Good job it looks already better than the current openScale icon :+1: 

I think the design of the new icon is a bit too "complex". For example in a small icon scale you can't really see the black dots anymore. 

Some app icons maybe for inspiration which I like are:
https://play.google.com/store/apps/details?id=jp.co.recruit.mtl.beslim
https://play.google.com/store/apps/details?id=com.yunmai.scaleen
https://play.google.com/store/apps/details?id=com.ikdong.weight
https://play.google.com/store/apps/details?id=com.dencreak.weightwar
https://play.google.com/store/apps/details?id=org.fmouri.weightLog
https://play.google.com/store/apps/details?id=com.medm.app.health
https://play.google.com/store/apps/details?id=com.despdev.weight_loss_calculator

But it is definitely only my personal opinion. I think often in design less is more. But I really like the shadow play in the new icon.

**Comment by DavidXochipilli** [comment]
I will send two icon versions, one with the dots and one without.
Is that fine or do you have other ideas? 🤔

**Comment by oliexdev** [comment]
I don't have really a good idea. Maybe the icon designer has another idea or somebody else in the openScale community?

But anyway I will use also the icon version above if nobody has another idea. But there is also no time pressure because I will be in the next weeks anyway on my holidays and travel around the world :smile:

**Comment by DavidXochipilli** [comment]
@Sajidshaik and I don't have another good idea either. 😥
He will upload a .zip with your requested sizes and the .ai source file in some days.
Maybe the community can give us some input, they are our last hope. 😀
We wish you an awesome journey! 😎

**Comment by oliexdev** [comment]
Thank you!

I created a icon based on https://www.flaticon.com/free-icon/weight_527064 by Those Icons

What do you think?

![openscale_new](https://user-images.githubusercontent.com/10166185/29930290-4fb7c7aa-8e6e-11e7-8b01-97dac1308698.png)

**Comment by DavidXochipilli** [comment]
To be honest, I don't like your icon that much because it isn't really Material. 🤔

**Comment by nykroy** [comment]
I think the same that @DavidXochipilli

**Comment by DavidXochipilli** [comment]
I attached a zip with the icon.
For the [Credits](https://github.com/oliexdev/openScale/blob/master/CREDITS): [Sajid Shaik](https://plus.google.com/u/0/+DroidAppsReviewer).
[OpenScale.zip](https://github.com/oliexdev/openScale/files/1308578/OpenScale.zip)

**Comment by Sajidshaik** [comment]
Here you go @oliexdev 
[OpenScale.zip](https://github.com/oliexdev/openScale/files/1308882/OpenScale.zip)

**Comment by oliexdev** [comment]
I am back from my journey, I had a great time :sunglasses: 

Thank you so much for your designed icon @Sajidshaik and @DavidXochipilli for your effort but honestly currently I am a bit uncertain which icon I should use... :thinking: 

What do you mean it isn't really material @DavidXochipilli?

**Comment by oliexdev** [comment]
Btw @Sajidshaik under which license is your icon?

**Comment by DavidXochipilli** [comment]
> I am back from my journey, I had a great time 😎

That are great news! 👍

> Product icon design is inspired by the tactile and physical quality of material. Each icon is cut, folded, and lit as paper would be, but represented by simple graphic elements. The quality of the material is sturdy, with clean folds and crisp edges. The matte-like finish interacts with light through subtle highlights and consistent shadows.

[https://material.io/guidelines/](https://material.io/guidelines/)
[https://material.io/guidelines/style/icons.html](https://material.io/guidelines/style/icons.html)
[Making Material Design on YouTube](https://youtu.be/rrT6v5sOwJg)

Maybe @Sajidshaik can add a better explanation. It is quite difficult to explain for me. 🤔
Maybe even [Google](https://www.youtube.com/channel/UClKO7be7O9cUGL94PHnAeOA) does not know it exactly. 🤤

**Comment by oliexdev** [comment]
I understand what you mean. I've now decided on an icon, I think the below icon fits more to openScale because of its simplicity but that doesn't mean Sajidshaik icon is bad. Hope your are not mad at me... :(

I have changed the icon a little bit but to comply with the material design guide, I think I am too bad at inkscape. 

![ic_launcher_openscale](https://user-images.githubusercontent.com/10166185/30774927-71000b24-a08a-11e7-8c21-0acb4255e417.png)

svg-file

[launcher_openscale.svg.zip](https://github.com/oliexdev/openScale/files/1326659/launcher_openscale.svg.zip)
